integrations/gcp/.port/spec.yaml

```diff
@@ -0,0 +1,27 @@
+type: gcp
+description: gcp integration for Port Ocean
+icon: GoogleCloud
+features:
+  - type: exporter
+    section: Cloud Providers
+configurations:
+  - name: parent
+    required: true
+    type: string
+    description: The Organization/Folder/Project you want the integration to fetch resources from.
```

Comment:
why not fetch from everything you have permissions to?

Reply:
but wont i get only projects i have access to?
tehy can simply give me less permissions